src/Commands/Self/Plugin/InstallCommand.php

```diff
@@ -42,6 +43,38 @@ public function install(array $projects)
         }
     }
 
+    /**
+     * Migrate Terminus 2 plugins.
+     *
+     * @command self:plugin:migrate
+     * @aliases plugin:migrate
+     *
+     * @throws \Pantheon\Terminus\Exceptions\TerminusException
+     */
+    public function migrate()
+    {
+        $default_plugins_dir = getenv('HOME') . DIRECTORY_SEPARATOR . '.terminus' . DIRECTORY_SEPARATOR . 'plugins';
+        $plugins_dir = !empty(getenv('TERMINUS2_PLUGINS_DIR')) ? getenv('TERMINUS2_PLUGINS_DIR') : $default_plugins_dir;
+        if (is_dir($plugins_dir)) {
+            $plugin_dirs = glob($plugins_dir . DIRECTORY_SEPARATOR . '*', GLOB_ONLYDIR);
+            if (!empty($plugin_dirs)) {
+                $projects = [];
+                foreach ($plugin_dirs as $dir) {
+                    $projects[] = $this->getProjectNameFromPath($dir);
+                }
+                if (!empty($projects)) {
+                    $this->install($projects);
```

Comment:
You could get rid of a bunch of `else` statements if you return after installing, and log NO _PLUGINS at the bottom of this routine.

Reply:
Done.